src/tools/binaryen-shell.cpp

```diff
@@ -77,8 +77,8 @@ static void run_asserts(size_t* i, bool* checked, Module* wasm,
   std::unique_ptr<ShellExternalInterface> interface;
   std::unique_ptr<ModuleInstance> instance;
   if (wasm) {
-    interface = make_unique<ShellExternalInterface>();
-    instance = make_unique<ModuleInstance>(*wasm, interface.get());
+    interface = wasm::make_unique<ShellExternalInterface>();
```

Comment:
I'm fine with making this `wasm::make_unique`. IIRC, `std::make_unique` is a new addition in C++14. Does VS2015 default to C++14?